release/release-notes/odfe-data-prepper.release-notes-1.0.0.md

```diff
@@ -0,0 +1,25 @@
+# 2021-05-xx Version 1.0.0
+
+## Highlights
+* Now builds using version 1.0+ of the OpenTelemetry tracing specification
+* Additional TraceGroup fields are emitted for enhanced searching and filtering with the Trace Analytics Kibana plugin
+* An OpenSearch-compatible version of Data Prepper has been released under the [opensearch-project repo](https://github.com/opensearch-project/data-prepper/releases)
+
+## Compatibility
+### Issue #1
+This release is intended to be used by version 1.13.2.1+ of the Trace Analytics Kibana plugin. The most recent Open Distro for Elasticsearch bundle contains version 1.13.2.0, so only use this version of Data Prepper if you have manually upgraded the Trace Analytics plugin.
+
+
+### Issue #2
+This release is backwards incompatible with data emitted by alpha/beta Data Prepper versions.
+
+If your Elasticsearch cluster has ingested data from a previous version of Data Prepper, you will need to delete the data before running this release of Data Prepper by:
+1. Stopping any existing instances of Data Prepper
+2. Deleting the span and service-map indices. This can be done in the Kibana UI by navigating to Dev Tools via the left sidebar, then running the following commands:
+   1. DELETE /otel-v1-apm-span-*
+   2. DELETE /otel-v1-apm-service-map
+   3. DELETE /_template/otel-v1-apm-span-index-template
```

Comment:
Also delete the raw span policy created in previous versions:

```
DELETE _opendistro/_ism/policies/raw-span-policy
```

Reply:
I've tested migrating Data Prepper versions without deleting the span policy and it seems to work fine. Any reason not deleting it would cause an issue?

Reply:
@wrijeff Ah. Maybe it is optional for ODFE < 1.13.0. I had thought about the possibility that cx created policy in ODFE 1.13+ through previous data-prepper version, which failed to attach to the raw index due to the old ISM API (#546 ). We might need to add a note about it. The solution is either:

(1) attach the policy to raw span index pattern manually in the kibana index management UI. Assume it is do-able.
(2) Just delete the raw-span-policy and let the new data-prepper recreate it and attach it.